SpywareFilter/sections/specific.txt

```diff
@@ -22,6 +22,8 @@
 ||nd.demdex.net/event?d_mid=$redirect=nooptext,important,domain=9now.com.au
 ||adc.nine.com.au/?token=$redirect=nooptext,important,domain=9now.com.au
 !
+||seoul.co.kr/weblog/proc/weblog_*.php
+||seoul.co.kr/weblog/proc/weblog_seoul*.js
```

Comment:
why not `||seoul.co.kr/weblog/proc/weblog_`?

Reply:
Okay, b4cd5138ef600e1e9a31388f6e296c538671e410

Reply:
Or even `||seoul.co.kr/weblog/`? I see this is in EasyPrivacy. Can this cause trouble?

Reply:
4d1124c0ff60dc7ad3cc205110053a78539323d4